styles.css

```diff
@@ -0,0 +1,391 @@
+*{
+	box-sizing: border-box;
```

Comment:
Whoops! Watch your indentation for this whole file. It's a a little wonky across the board